nss-day-cohort-41/react-nutshell-the-cashews
# Th friends add

# Description
Added Friends functionality of being able to click on a button and open a modal, then searching for friends, and then being able to add friends that are available.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
- [] Bug fix (non-breaking change which fixes an issue)
# Testing Instructions
Open the dashboard and click on the add friend button on the top left. A modal should appear with a searchbar. Click on the searchbar and type in who you want to find and the results should update with each keypress. You should be able to add a friend and when you close the modal they should be in your friends list.
# Checklist:
